### PR TITLE
chore(release notes): Updated release notes for 2020.14

### LINKED ISSUE #5063: Release notes 2020.14

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5063

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,115 @@ section: get-started
...
+- **Notification drawer:** 
+  - Added tooltips to truncated titles ([#5043](https://github.com/patternfly/patternfly-react/pull/5043))
+- **Page:** 
+  - Updated for page section enhancements ([#5025](https://github.com/patternfly/patternfly-react/pull/5025))
+- **Pagination:** 
+  - Added sticky modifier flag ([#4977](https://github.com/patternfly/patternfly-react/pull/4977))
+- **Select:** 
+  - Updated to display label and not value in typeahead ([#4916](https://github.com/patternfly/patternfly-react/pull/4916))
+  - Fixed to skip the divider children in getDisplay ([#4998](https://github.com/patternfly/patternfly-react/pull/4998))
+  - Updated to allow custom chip render props in the multi typehead select . ([#5031](https://github.com/patternfly/patternfly-react/pull/5031))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:68]: Stray `.`

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,115 @@ section: get-started
...
+  - Updated to export WizardContext  ([#4933](https://github.com/patternfly/patternfly-react/pull/4933))
+
+### Catalog view extension
+- Fixed example css selectors ([#4966](https://github.com/patternfly/patternfly-react/pull/4966))
+
+### Chart
+- Fixed example button location ([#4993](https://github.com/patternfly/patternfly-react/pull/4993))
+- Fixed legend example size ([#4995](https://github.com/patternfly/patternfly-react/pull/4995))
+- Fixed stack chart example size ([#4994](https://github.com/patternfly/patternfly-react/pull/4994))
+- **Charts:** Legend tooltip example ([#5006](https://github.com/patternfly/patternfly-react/pull/5006))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:91]: Remove `**Charts:** `

**Review by redallen — CHANGES_REQUESTED** [review]
Nits

**Review by redallen — APPROVED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,115 @@ section: get-started
...
+- **Notification drawer:** 
+  - Added tooltips to truncated titles ([#5043](https://github.com/patternfly/patternfly-react/pull/5043))
+- **Page:** 
+  - Updated for page section enhancements ([#5025](https://github.com/patternfly/patternfly-react/pull/5025))
+- **Pagination:** 
+  - Added sticky modifier flag ([#4977](https://github.com/patternfly/patternfly-react/pull/4977))
+- **Select:** 
+  - Updated to display label and not value in typeahead ([#4916](https://github.com/patternfly/patternfly-react/pull/4916))
+  - Fixed to skip the divider children in getDisplay ([#4998](https://github.com/patternfly/patternfly-react/pull/4998))
+  - Updated to allow custom chip render props in the multi typehead select ([#5031](https://github.com/patternfly/patternfly-react/pull/5031))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:68]:
```suggestion
  - Updated to allow custom chip render props in the multi typeahead select ([#5031](https://github.com/patternfly/patternfly-react/pull/5031))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,115 @@ section: get-started
...
+- **Button:** 
+  - Fixed sandbox for button examples with tooltips ([#5033](https://github.com/patternfly/patternfly-react/pull/5033))
+- **Context selector:** 
+  - Added footer to selector ([#4976](https://github.com/patternfly/patternfly-react/pull/4976))
+- **Data list:**
+  - Added documentation for properties of DataListDragButton ([#5047](https://github.com/patternfly/patternfly-react/pull/5047))
+  - Added expand/collapse all ([#5034](https://github.com/patternfly/patternfly-react/pull/5034))
+  - Exposed grid modifiers as properties ([#4969](https://github.com/patternfly/patternfly-react/pull/4969))
+  - Fixed to cancel moving for invalid drops ([#4996](https://github.com/patternfly/patternfly-react/pull/4996))
+  - Converted DataLIstAction to functional component ([#4947](https://github.com/patternfly/patternfly-react/pull/4947))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:34]:
```suggestion
  - Converted DataListAction to functional component ([#4947](https://github.com/patternfly/patternfly-react/pull/4947))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
2 very small nits, otherwise looks good

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Review by redallen — APPROVED** [review]

**Pull request merged**